troubleshooting/known-issues.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: Known Issues
 parent_title: Troubleshooting
 permalink: /troubleshooting/known-issues.html
-modification_time: 2016-25-06T07:30:29+00:00
+modification_time: 2016-06-25T07:30:29+00:00
```

Comment:
Why this particular date?

Reply:
The month and day were swapped. The renderer of Github Pages will throws an error on this wrong format.